doc/angus-tutorial.md

```diff
@@ -0,0 +1,575 @@
+# Quick Insights from Sequencing Data with sourmash
+
+## Getting started
```

Comment:
I think remove the entire Getting Started section, since it's ANGUS specific and no one will know what ANGUS is :)

have you/can you try running through this tutorial on farm?